CODEOWNERS

```diff
@@ -20,23 +20,18 @@
 # on changes to PIL code-generator
 /bb-pilcom @Maddiaa0 @jeanmon @IlyasRidhuan @fcarreiro
 # on changes to PIL code (AVM circuit)
-/barretenberg/cpp/pil @Maddiaa0 @jeanmon @IlyasRidhuan @fcarreiro
+/barretenberg/cpp/pil @jeanmon @IlyasRidhuan @fcarreiro
 # on changes to PIL-generated C++
 /barretenberg/cpp/src/barretenberg/**/generated @jeanmon @IlyasRidhuan @fcarreiro
```

Comment:
I think we don't have this one anymore (it's under avm and vm2?). Could you check?